Small fixes (#16)
ScilifelabDataCentre/dds_workshop: docs/2_getting_started_session.md

This link is to register a new account, in the Test instance and in a custom unit for this workshop. Open the link and create a new account.

> To search for this for email, which has being delivered this week, you can filter with the follwing sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`

Reviewer: ```suggestion
To search for this for email, which has bee sent earlier this week, you can search your mailbox for the following sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`
```

> To search for this for email, which has being delivered this week, you can filter with the follwing sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`

Reviewer 2: Can we leave this as it was (but fixing the mixed up word of with/which)? I think the first sentence phrasing was better. 

```suggestion
Search for an email, which has being delivered this week, which contains the sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`
```

Author: The issue was that it was written imperative, and could lead to confusion because below we ask them to do this.

> To search for this for email, which has been sent earlier this week, you can search your mailbox for the following sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`

Reviewer 2: ```suggestion
To search for the email, which was sent earlier this week, search your mailbox for the following sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`
```

To search for this for email, which has being delivered this week, you can filter with the follwing sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`

![Screenshot of searching for the invitation email](https://i.imgur.com/KpCbO0U.png)

![Screenshot of the email text](https://i.imgur.com/jNRslTk.png)

> After finding the email, click on the link and follow the instruction to register a new account.

Reviewer 2: I also think this is useful so would not remove?

Author: Yeah, I just thought that it could be maybe too obvious that you need to fill the boxes to register

> Once that is done. The command to log in is:

Reviewer: ```suggestion
Once that is done, the command to log in with the client is:
```

This link is to register a new account, in the Test instance and in a custom unit for this workshop. Open the link and create a new account.

> To search for the email, which was sent earlier this week, search your mailbox for the following sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`

Reviewer: maybe this is a good formulation:
```suggestion
The email, which was sent earlier this week, can be found in your mailbox by searching for the following sentence: `You have been invited to join the SciLifeLab Data Delivery System (DDS)`
```

After completing authentication, dds-cli will automatically save an authentication token file (.dds_cli_token) by default in your home directory. You have the possibility to specify the location and the filename of the token file, but we will explore this functionality in the end of this session.

 - [ ] TASK: Find the email and create an account. Make sure your password is a strong one.

>  - [ ] TASK: Login using your information. Note that the password will not be shown while you are typing it. 

Reviewer: ```suggestion
 - [ ] TASK: Log in using your information. Note that the password will not be shown while you are typing it. 
```
